calicoctl/resourcemgr/globalnetworkset.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2018 Tigera, Inc. All rights reserved.
```

Comment:
it will be nice if we also have an example GlobalNetworkSet in: https://github.com/projectcalico/calicoctl/tree/master/test-data/v3 since we use it for manual testing and as a reference